carma_wm_ctrl/src/WMBroadcaster.cpp

```diff
@@ -162,7 +162,7 @@ std::shared_ptr<Geofence> WMBroadcaster::geofenceFromMsg(const cav_msgs::Traffic
     addPassingControlLineFromMsg(gf_ptr, msg_v01, affected_llts);
   }
 
- if (msg_detail.choice == cav_msgs::TrafficControlDetail::CLOSED_CHOICE || msg_detail.choice == cav_msgs::TrafficControlDetail::CLOSED)
+ if (msg_detail.choice == cav_msgs::TrafficControlDetail::CLOSED_CHOICE && msg_detail.choice == cav_msgs::TrafficControlDetail::CLOSED)
```

Comment:
Please see the last part closely. it is a little different. You can copy paste below directly
msg_detail.choice == cav_msgs::TrafficControlDetail::CLOSED_CHOICE && msg_detail.closed==cav_msgs::TrafficControlDetail::CLOSED